packages/uploadthing/test/__test-helpers.ts

```diff
@@ -211,7 +201,7 @@ export const it = itBase.extend({
 
           // Simulate polling - at least once
           yield HttpResponse.json({ status: "still waiting" });
-          if (!file) {
+          while (!file) {
```

Comment:
polling can fail after first attempt, keep retrying until we have a file in db (or test times outs and fails)